lib/Tapable.js

```diff
@@ -53,33 +53,37 @@ Tapable.mixin = function mixinTapable(pt) {
 	copyProperties(Tapable.prototype, pt);
 };
 
-Tapable.prototype.applyPlugins = function applyPlugins(name) {
+Tapable.prototype.applyPlugins = function applyPlugins(name, param1, param2) {
 	if(!this._plugins[name]) return;
```

Comment:
So I was curious about Chrome Stable (V8 6.0.286.52, same as Node 8.3, and this code is running on Node) and the results are bit different:

![screen shot 2017-08-13 at 09 15 45](https://user-images.githubusercontent.com/5106466/29246970-17aab8b4-8008-11e7-8e6d-8b4f96e5c372.png)